Spigot-Server-Patches/0628-Guardian-beam-workaround.patch

```diff
@@ -0,0 +1,35 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Gabscap <git@gabscap.de>
+Date: Sat, 19 Mar 2016 22:25:11 +0100
+Subject: [PATCH] Guardian beam workaround
+
+This patch is a workaround for MC-165595
+
+diff --git a/src/main/java/net/minecraft/server/PacketPlayOutUpdateTime.java b/src/main/java/net/minecraft/server/PacketPlayOutUpdateTime.java
+index 88c3d7efaf467c1c1487f589c2cdbfb6aba734ec..15af5927f3d1d41b2a4f86b4267aa6a0485fb53e 100644
+--- a/src/main/java/net/minecraft/server/PacketPlayOutUpdateTime.java
++++ b/src/main/java/net/minecraft/server/PacketPlayOutUpdateTime.java
+@@ -4,7 +4,13 @@ import java.io.IOException;
+ 
+ public class PacketPlayOutUpdateTime implements Packet<PacketListenerPlayOut> {
+ 
++    // World Age in ticks
++    // Not changed by server commands
++    // World Age must not be negative
+     private long a;
++    // Time of Day in ticks
++    // If negative the sun will stop moving at the Math.abs of the time
++    // Displayed in the debug screen (F3)
+     private long b;
+ 
+     public PacketPlayOutUpdateTime() {}
+@@ -19,6 +25,9 @@ public class PacketPlayOutUpdateTime implements Packet<PacketListenerPlayOut> {
+             }
+         }
+ 
++        // Paper start
++        this.a = this.a % 192000;
++        // Paper end
```

Comment:
Needs OBFHelper + just use a single comment for one line change

Reply:
This was just to re-add this https://github.com/PaperMC/Paper/pull/4786/files#diff-a6654f73ef4e224bdbf6d518be3c8a174aad5935c7bac049018f1a268a0234ab that got removed, thought the patch it self was fine since it was added in the first place.

Reply:
its _old_, we werent so good at handling that kinda stuff back then ^^

we can fix up that after proval tho